### DISCUSSION TITLE
Preset to group Python packages in a single pr

### DISCUSSION THREAD

**Discussion by AliyevH** [renovatebot/renovate#27610] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.191

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### What is your question?

If we have 3 PRs, after merging 1 of them, it is impossible to merge af others because of conflicts. I have to rebase every PR.
That is why I wanted to group all poetry updates in a single PR.
Could not found ready to use preset for it. If there is preset please tell me how to use.
If not can we create new preset? I would like to contribute and add preset to renovate.

{
    "description": "Group Python packages together.",
    "groupName": "All Python",
    "groupSlug": "all",
    "matchDatasources": [
      "pypi"
    ],
},

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
We don't want to create a preset for every possible datasource when it's so easy for you to create a package rule like above

**Reply by AliyevH** [reply]
Thanks a lot.
